src/sys/wait.rs

```diff
@@ -267,13 +253,14 @@ impl From<i32> for PidGroup {
     }
 }
 
-impl Into<i32> for PidGroup {
-    fn into(self) -> i32 {
+impl Into<pid_t> for PidGroup {
+    fn into(self) -> pid_t {
         match self {
-            PidGroup::ProcessID(pid)      => pid as i32,
-            PidGroup::ProcessGroupID(pid) => -(pid as i32),
-            PidGroup::AnyGroupChild       => 0,
-            PidGroup::AnyChild            => -1
+            PidGroup::ProcessID(pid) if pid < Pid::from_raw(-1)      => -pid_t::from(pid),
```

Comment:
Please don't align the `=>` in match arms, just have them surrounded by one space on each size.